source/world/level/Level.cpp

```diff
@@ -769,19 +770,20 @@ Player* Level::getNearestPlayer(float x, float y, float z, float maxDist) const
 	return pPlayer;
 }
 
-bool Level::containsFireTile(AABB aabb)
+bool Level::containsFireTile(const AABB& aabb)
 {
-	int minX = Mth::floor(aabb.min.x),     minY = Mth::floor(aabb.min.y),     minZ = Mth::floor(aabb.min.z),
-		maxX = Mth::floor(aabb.max.x + 1), maxY = Mth::floor(aabb.max.y + 1), maxZ = Mth::floor(aabb.max.z + 1);
+	TilePos min(aabb.min),
+		    max(aabb.max + 1);
```

Comment:
+TilePos(1) or +TilePos(1,1,1).  Mentioned this before but you should NOT have implicit constructors like that.